extensions/widgets/svgpath/tests/properties.livecodescript

```diff
@@ -0,0 +1,68 @@
+﻿script "SvgPathWidgetProperties"
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+private function getPathData
+	return "M1664 698Q1664 720 1638 746L1275 1100 1361 1600Q1362 1607 1362 1620 1362 1641 1351.5 1655.5 1341 1670 1321 1670 1302 1670 1281 1658L832 1422 383 1658Q361 1670 343 1670 322 1670 311.5 1655.5 301 1641 301 1620 301 1614 303 1600L389 1100 25 746Q0 719 0 698 0 661 56 652L558 579 783 124Q802 83 832 83 862 83 881 124L1106 579 1608 652Q1664 661 1664 698Z"
+end getPathData
+
+on TestSetup
+	TestLoadExtension "com.livecode.library.widgetutils"
+	TestLoadExtension "com.livecode.library.iconSVG"
+	TestLoadExtension "com.livecode.widget.svgPath"
```

Comment:
@hwbehrens The `TestLoadExtension` command currently is case-sensitive on Linux -- the argument has to match the name specified in the widget's LCB `widget` declaration exactly.  You need to change `svgPath` to `svgpath`.